_docs/development/environment.md

```diff
@@ -179,16 +197,15 @@ bundle install
 The API is a Rails application, and you'll use standard Rails commands to manage the database. The following commands create the database, loads the schema, and add seed data.
 
 ```
-cd ~/src/manifold/api
 rails db:create
 rails db:schema:load
 rails db:seed
 ```
 
-While you're at it, go ahead and create an admin user. Be sure to replace the values in the command below with your email, password, first name, and last name.
+While you're at it, go ahead and create an admin user. Be sure to replace the values in the command below with your email, password, first name, and last name. For bash users, the backslash characters are not needed before the square brackets.
 
 ```
-rails manifold:user:create:admin["email@email.com","password","FirstName","LastName"]
+rails manifold:user:create:admin\["email@email.com","password","FirstName","LastName"\]
```

Comment:
Here is the only other zsh vs bash difference. I believe that the backslash version will run on both zsh and bash, but I have not tested it on bash (so I made a note in the description above).